distribution/src/conf/deployment.toml

```diff
@@ -15,3 +17,144 @@ file_name = "client-truststore.jks"
 password = "wso2carbon"
 alias = "symmetric.key.value"
 algorithm = "AES"
+
+# [message_formatters]
+# form_urlencoded =  "org.apache.synapse.commons.formatters.XFormURLEncodedFormatter"
+# multipart_form_data =  "org.apache.axis2.transport.http.MultipartFormDataFormatter"
+# application_xml = "org.apache.axis2.transport.http.ApplicationXMLFormatter"
+# text_xml = "org.apache.axis2.transport.http.SOAPMessageFormatter"
+# soap_xml = "org.apache.axis2.transport.http.SOAPMessageFormatter"
+# text_plain = "org.apache.axis2.format.PlainTextFormatter"
+# application_json =  "org.wso2.micro.integrator.core.json.JsonStreamFormatter"
+# json_badgerfish = "org.apache.axis2.json.JSONBadgerfishMessageFormatter"
+# text_javascript = "org.apache.axis2.json.JSONMessageFormatter"
+# octet_stream = "org.wso2.carbon.relay.ExpandingMessageFormatter"
+# application_binary =  "org.apache.axis2.format.BinaryFormatter"
+
+# [message_builders]
+# application_xml = "org.apache.axis2.builder.ApplicationXMLBuilder"
+# form_urlencoded = "org.apache.synapse.commons.builders.XFormURLEncodedBuilder"
+# multipart_form_data = "org.apache.axis2.builder.MultipartFormDataBuilder"
+# text_plain = "org.apache.axis2.format.PlainTextBuilder"
+# application_json = "org.wso2.micro.integrator.core.json.JsonStreamBuilder"
+# json_badgerfish = "org.apache.axis2.json.JSONBadgerfishOMBuilder"
+# text_javascript = "org.apache.axis2.json.JSONBuilder"
+# octet_stream =  "org.wso2.carbon.relay.BinaryRelayBuilder"
+# application_binary = "org.apache.axis2.format.BinaryBuilder"
+
+# [[custom_message_formatters]]
+# content_type = text/html"
+# class = "org.wso2.carbon.relay.ExpandingMessageFormatter"
+
+# [[custom_message_builders]]
+# content_type = "text/html"
+# class = "org.wso2.carbon.relay.BinaryRelayBuilder"
+
+# [blocking.message_formatters]
+# octet_stream = "org.wso2.carbon.relay.ExpandingMessageFormatter"
+
+# [blocking.message_builders]
+# octet_stream =  "org.wso2.carbon.relay.BinaryRelayBuilder"
+
+# [[blocking.custom_message_formatters]]
+# content_type = "text/html"
+# class = "org.wso2.carbon.relay.ExpandingMessageFormatter"
+
+# [[blocking.custom_message_builders]]
+# content_type = "text/html"
+# class = "org.wso2.carbon.relay.BinaryRelayBuilder"
+
+# [transport]
+# default_content_type = "empty/content"
+
+# [transport.http]
+# socket_timeout = 180000   # timeout in milliseconds
```

Comment:
shall we add all the passthru properties. I feel like we need them.

Reply:
isn't that too much

Reply:
added essential params in https://github.com/wso2/micro-integrator/pull/1689/commits/c07b184479b0870e01d6735515788d4061692bd1